stubs/pyjks/jks/jks.pyi

```diff
@@ -47,7 +47,7 @@ class PrivateKeyEntry(AbstractKeystoreEntry):
         self,
         *,
         cert_chain: list[tuple[_CertType, bytes]] = ...,
-        encrypted: bool = ...,
+        encrypted: bytes | None = ...,
```

Comment:
```suggestion
        encrypted: bytes | None = None,
```

(Same for some of the other defaults. Their types are also missing `None`, see above. But changing this in this PR is optional.)